UPGRADING.md

```diff
@@ -0,0 +1,179 @@
+
+## 1. Migrate from webpacker to jsbundling-rails
+
+Start by following these steps:
+
+1. Replace `webpacker` with `jsbundling-rails` in your Gemfile;
+2. Run `./bin/bundle install`;
+3. Run `./bin/rails javascript:install:webpack`;
+4. Remove the file `config/initializers/assets.rb`;
+5. Remove the file `bin/webpack`;
+5. Remove the file `bin/webpack-dev-server`;
+6. Remove the folder `config/webpack`;
+7. Replace all instances of `javascript_pack_tag` with `javascript_include_tag` and add `defer: true` to them.
+
+After you are done you will notice that the install step added various files to your project and updated some of the existing ones.
+
+**The new 'bin/dev' and 'Procfile.dev' files**
+
+The `./bin/dev` file is a shell script that uses [foreman](https://github.com/ddollar/foreman) and `Procfile.dev` to start two processes in a single terminal: `rails s` and `yarn build`. The latter replaces `webpack-dev-server` in bundling and watching for changes in javascript files.
+
+**The 'build' attribute added to packages.json**
+
+This is the command that `yarn build` will use to bundle javascript files.
+
+**The new 'webpack.config.js file'**
+
+In `webpacker` this file was hidden inside the gem, but now you can edit it directly. If you had custom configuration in `config/webpack` you can move them to here. Projects with multiple entrypoints will need to adjust the `entry` attribute:
+
+```js
+module.exports = {
+  entry: {
+    application: "./app/javascript/application.js",
+    admin: "./app/javascript/admin.js"
+  }
+}
+```
+
+**The 'link_tree' directive added to 'app/assets/manifest.js'**
+
+This tells Sprockets to include the files in `app/assets/builds` during `assets:precompile`. This is the folder where `yarn build` will place the bundled files, so make sure you commit it to the repository and don't delete it when cleaning assets. 
+
+**What about babel?**
+
+If you would like to continue using babel for transpiling, you will need to configure it manually. First, open `webpack.config.js` and add this:
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(js)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader']
+      }
+    ]
+  }
+}
+```
+
+Then open `packages.json` and add this:
+```json
+"babel": {
+  "presets": [
+    "./webpack.babel.js"
+  ]
+}
+```
+
+Finally, download [webpackers babel preset](https://github.com/rails/webpacker/blob/master/package/babel/preset.js) file and place it in the same directory as `packages.json` with the name `webpack.babel.js`.
+
+## 2. Migrate from sass-rails to cssbundling-rails
+
+Start by following these steps:
+
+1. Add `cssbundling-rails` to your Gemfile (do not remove `sass-rails`, Sprockets still needs it);
```

Comment:
I don't think this is true anymore sass compilation seems to work fine without it